monitor/penalty.go

```diff
@@ -95,3 +103,34 @@ func (m *Monitor) submitPenalty(penaltyEvent PenaltyEvent) {
 	}
 	log.Infoln("Slash tx submitted", tx.Hash().Hex())
 }
+
+func (m *Monitor) validatePenaltySigs(nonce uint64) bool {
+	penalty, err := slash.CLIQueryPenalty(m.Transactor.CliCtx, slash.StoreKey, nonce)
```

Comment:
Duplicated query with line 69. 
Actually it seems that the`CLIQueryPenaltyRequest` function itself is also uncessary